src/spec/vk.xml

```diff
@@ -506,7 +506,7 @@ maintained in the master branch of the Khronos Vulkan Github project.
             <member optional="true">const <type>VkPhysicalDeviceFeatures</type>* <name>pEnabledFeatures</name></member>
             <validity>
                 <usage>pname:ppEnabledLayerNames must: either be sname:NULL or contain the same sequence of layer names that was enabled when creating the parent instance</usage>
-                <usage>Any given element of pname:ppEnabledExtensionNames must: be the name of an extension present on the system, exactly matching a string returned in the sname:VkExtensionProperties structure by fname:vkEnumerateDeviceExtensionProperties</usage>
+                <usage>Each extension name in the pname:ppEnabledExtensionNames array must be one of those provided by the implementation or by an enabled layer and must exactly match a string returned in the sname:VkExtensionProperties structure from flink:vkEnumerateDeviceExtensionProperties</usage>
                 <usage>If an extension listed in pname:ppEnabledExtensionNames is provided as part of a layer, then both the layer and extension must: be enabled to enable that extension</usage>
```

Comment:
This was my original issue #214 Should it be removed now with the changes in place?